src/main/components/urlBar.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState } from 'react';
+
+const UrlBar = (props) => {
+  const method = (props.AorB === 'B' ? 'POST' : 'GET')
+  const [selected, setSelected] = useState(method);
+
+  function sendFetch () {
+    const sendingObj = { method: selected, mode: 'cors' };
+    if (props.SourceOrDest === 'dest') sendingObj.body = JSON.stringify(props.requestBody);
```

Comment:
We’ll send a separate fetch request for _each_ version of mocked up data. What’s the plan for queuing up multiple requests to the destination? Seems ok for this PR if we leave as is, so long as we’re on the same page—we’ll have to refactor later to handle queuing multiple requests.